src/main/java/RacingGame/model/Car.java

```diff
@@ -0,0 +1,57 @@
+package RacingGame.model;
+
+public class Car {
+    private static final String CARNAME_LENGTH_EXCEED = "차량의 이름은 5글자가 넘어갈 수 없습니다";
+    private static final String DEFAULT_CAR_NAME = "NULL";
+    private static final int DEFAULT_MOVE_DISTANCE = 1;
+    private static final int INITAIL_CAR_POSITION = 0;
+    private final String name;
+    private int position;
+
+    public Car(String name, int position) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException(CARNAME_LENGTH_EXCEED);
+        }
+
+        this.name = name;
+        this.position = position;
+    }
+
+    public Car() {
+        this(DEFAULT_CAR_NAME, INITAIL_CAR_POSITION);
+    }
+
+    public Car(int position) {
+        this(DEFAULT_CAR_NAME, position);
+    }
+
+    public Car(String name) {
+        this(name, INITAIL_CAR_POSITION);
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void move(MovableStrategy strategy) {
+        if (isMoveCar(strategy)) {
```

Comment:
```suggestion
        if (strategy.movable())) {
```

전략을 매개변수로 받았다는것으로 전진전략의 책임을 전략에게 위임하기에 여기서 함수로 분리까지 할 필요는 없어보입니다!